BuildMasterAutomation/Functions/Publish-BMReleasePackage.ps1

```diff
@@ -16,11 +16,16 @@ function Publish-BMReleasePackage
     Publish-BMReleasePackage -Session $session -Package $package
 
     Demonstrates how to deploy a package by passing a package object to the `Package` parameter. This object must have an `id` or `pipeline_id` property.
-
+    
     .EXAMPLE
     Publish-BMReleasePackage -Session $session -Package 383
 
     Demonstrates how to deploy a package by passing its ID to the `Package` parameter.
+
+    .EXAMPLE
+    Publish-BMReleasePackage -Session $session -Package $package

```

Comment:
Add the `-Stage` parameter to the example.

Reply:
Woops, at least I had changed the description.. :) Added..